modules/common/services/audio.nix

```diff
@@ -110,7 +163,7 @@ in
           StandardOutput = "journal";
           StandardError = "journal";
```

Comment:
Not 100% certain, but doesn't this already default to "journal"? Might be redundant
ref. https://www.freedesktop.org/software/systemd/man/latest/systemd.exec.html#StandardOutput=

Reply:
It seems you are right. Removed as redundant